README.md

```diff
@@ -58,3 +58,4 @@ The TAG has created the following working groups to investigate and discuss the
 | [Operator](https://github.com/cncf/tag-app-delivery/tree/main/operator-wg)     |                                                                        | Inactive                                                             |
 | [Artifacts](artifacts-wg/README.md)                                            | [artifacts-wg/README.md#chairs](./artifacts-wg/README.md#chairs)       | [artifacts-wg/README.md#meetings](./artifacts-wg/README.md#meetings) |
 | [App Development](app-development-wg/README.md)                                | [app-development-wg/README.md#chairs](./app-development-wg/README.md#chairs) | [app-development-wg/README.md#meetings](./app-development-wg/README.md#meetings) |
+| [Infrastructure Lifecycle](infra-lifecycle-wg/README.md)                                | [infra-lifecycle-wg/README.md#chairs](./infra-lifecycle-wg/README.md#chairs) | [infra-lifecycle-wg/README.md#meetings](./infra-lifecycle-wg/README.md#meetings) |
```

Comment:
💅 Would be a nice to have to keep the formatting for the table in line. So lining up the start of the 2nd and 3rd column in particular.

Reply:
Fixed